zebra-consensus/src/checkpoint.rs

```diff
@@ -198,7 +198,11 @@ impl CheckpointVerifier {
         // Find the height we want to start searching at
         let mut pending_height = match self.previous_checkpoint_height() {
             // Check if we have the genesis block as a special case, to simplify the loop
-            BeforeGenesis if !self.queued.contains_key(&BlockHeight(0)) => return WaitingForBlocks,
+            BeforeGenesis if !self.queued.contains_key(&BlockHeight(0)) => {
+                // XXX scratch tracing line for debugging, delete this
```

Comment:
did you want to delete this pre merge?

Reply:
I put it in to find the waiting-for-genesis issue so I'd like to remove it after we know that that part works.